samples/snippets/src/main/java/com/example/spanner/CreateInstanceWithAutoscalingConfigExample.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_create_instance_with_autoscaling_config]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.InstanceConfigId;
+import com.google.cloud.spanner.InstanceId;
+import com.google.cloud.spanner.InstanceInfo;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.spanner.admin.instance.v1.AutoscalingConfig;
+import com.google.spanner.admin.instance.v1.CreateInstanceMetadata;
+import java.util.concurrent.ExecutionException;
+
+class CreateInstanceWithAutoscalingConfigExample {
+
+  static void createInstance() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    createInstance(projectId, instanceId);
+  }
+
+  static void createInstance(String projectId, String instanceId) {
+    Spanner spanner = SpannerOptions.newBuilder().setProjectId(projectId).build().getService();
+    InstanceAdminClient instanceAdminClient = spanner.getInstanceAdminClient();
+
+    // Set Instance configuration.
+    String configId = "regional-us-central1";
+    // Create an autoscaling config.
+    AutoscalingConfig autoscalingConfig =
+        AutoscalingConfig.newBuilder()
+            .setAutoscalingLimits(
+                AutoscalingConfig.AutoscalingLimits.newBuilder().setMinNodes(1).setMaxNodes(2))
+            .setAutoscalingTargets(
+                AutoscalingConfig.AutoscalingTargets.newBuilder()
+                    .setHighPriorityCpuUtilizationPercent(65)
+                    .setStorageUtilizationPercent(95))
+            .build();
+    String displayName = "Descriptive name";
```

Comment:
We don't need a variable declaration here. Can use the value inline at `setDisplayName`

Reply:
That seems to be the way for create instance sample: https://github.com/googleapis/java-spanner/blob/main/samples/snippets/src/main/java/com/example/spanner/CreateInstanceExample.java#L40-L47 ? I'm ok with inlining it, just wonder if we need to be consistent for other createinstance samples?

Reply:
Updated to inline the display name.